src/features/customerList/__tests__/utils.test.ts

```diff
@@ -0,0 +1,153 @@
+import { getCustomersData } from '../utils';
+import { CUSTOMERS } from '../__generated__/CUSTOMERS';
+import { ContactMethod, ServiceType } from '../../../@types/__generated__/globalTypes';
+
+describe('utils', () => {
+  describe('getCustomersData', () => {
+    it('should convert data', () => {
```

Comment:
This test could perhaps be several smaller tests, to have a better insight to the reason if/once it starts failing

Reply:
Will split it.